qa/tasks/rbd_mirror_thrash.py

```diff
@@ -87,7 +86,7 @@ def _run(self):
             self.do_thrash()
         except Exception as e:
             # See _run exception comment for MDSThrasher
-            self.exception = e
+            super(RBDMirrorThrasher, self.__class__).exception.fset(self, e)
```

Comment:
I feel like this is needlessly complicated: why not just define a method in `Thrasher`:

````
def set_thrasher_exception(self, e):
    self._exception = e
````

and then here you can do:

`self.set_thrasher_exception(e)`

Manipulating the members of the parent class directly like this is a code smell IMO.